packages/react-native-skia/components/svg/SvgExamples/App.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Sample React Native App for react-native-svg library
+ * https://github.com/magicismight/react-native-svg/tree/master/Example
+ */
+'use strict';
+
+import React, {Component} from 'react';
+import { AppRegistry} from 'react-native';
+import {
+  Dimensions,
+  StyleSheet,
+  View,
+  Text,
+  ScrollView,
+} from 'react-native';
+
+import examples from './examples'
+var rowCount=1;
+const windowSize = Dimensions.get('window');
+
+const getExamples = () =>{
+  
+  var arr=[];
+  var arr1=[];
+  var appendedList=[];
+  for(var i=0;i<examples.length;i++) {
+    appendedList = appendedList.concat(examples[i].samples);
+  }
+   let data=appendedList.map((Samples)=>{
+       return (<View style={styles.innerContainer}>
+                  <Text style={{fontSize:'18',fontWeight:'bold', textAlign:'center' ,color:'#fff'}}>{Samples.title}</Text>
+                   <View style={styles.viewWrapper}>
+                     <Samples />
+                   </View>
+                </View>)
+   });
+  for(var j=0;j<data.length;j++) {
+    arr.push(<View style={styles.container}>
+              {data[j]}
+              </View>);
+    }
+   return arr;
```

Comment:
can be simplified, instead of 3 loops

Reply:
Will remove the last for loop, that is redundant now.